benchbuild/source/http.py

```diff
@@ -0,0 +1,70 @@
+"""
+Declare a http source.
+"""
+from typing import List, Mapping, Union
+
+import attr
+from plumbum import local
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'plumbum'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)